metrics/web/lib/common/presentation/metrics_theme/model/light_metrics_theme_data.dart

```diff
@@ -111,8 +111,10 @@ class LightMetricsThemeData extends MetricsThemeData {
             borderColor: _borderColor,
             hoverColor: _cardHoverColor,
             backgroundColor: scaffoldColor,
-            accentColor: ColorConfig.accentColor,
+            accentColor: ColorConfig.accentButtonColor,
+            accentHoverColor: ColorConfig.accentButtonHoverColor,
```

Comment:
In addition to the suggestion below, I think we can use the MetricsButtonThemeData for MetricsIconLabelButton. I've noticed that icon buttons have similar styles in the design.